cmd/apply.go

```diff
@@ -63,9 +63,10 @@ var applyCmd = &cobra.Command{
 	Use:   "apply [env-name] [-f <file-or-dir>]",
 	Short: `Apply local configuration to remote cluster`,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		if len(args) > 1 {
-			return fmt.Errorf("'apply' takes at most a single argument, that is the name of the environment")
+		if len(args) != 1 {
+			return fmt.Errorf("'apply' requires an environment name; use `env list` to see available environments")
 		}
+		env := args[0]
```

Comment:
There is a chance that args could be 0 since you only checking for != 1. This could panic.

Reply:
These should be ok, because the `!= 1` check will error out on all arg lengths not exactly 1, which is what we want. Can test using `ks apply` without args.